.github/workflows/unit-tests.yml

```diff
@@ -1,86 +1,86 @@
 name: Unit Tests
 
 on:
-  workflow_dispatch:
-  pull_request:
-    types: [opened, synchronize, reopened] # Default
+    workflow_dispatch:
```

Comment:
⬇️ : Why 4 spaces?

Other things that use two spaces:
1. All the Github actions docs
2. Prettier, by default
3. CircleCI docs
4. Ansible docs
5. Learn X in Y
6. This file before this PR (😏 )

Reply:
Let me revert. It was just what I have my default spacing set to on save.

Reply:
Merge it, it's fine.